recipes-dom0/dom0-image-rescue-initramfs/files/meta-xt-prod-extra/doc/local.conf.dom0-image-rescue-initramfs

```diff
@@ -0,0 +1,217 @@
+#
+# This file is your local configuration file and is where all local user settings
+# are placed. The comments in this file give some guide to the options a new user
+# to the system might want to change but pretty much any configuration option can
+# be set in this file. More adventurous users can look at local.conf.extended
+# which contains other examples of configuration which can be placed in this file
+# but new users likely won't need any of them initially.
+#
+# Lines starting with the '#' character are commented out and in some cases the
+# default values are provided as comments to show people example syntax. Enabling
+# the option is a question of removing the # character and making any change to the
+# variable as required.
+
+#
+# Machine Selection
+#
+# You need to select a specific machine to target the build with. There are a selection
+# of emulated machines available which can boot and run in the QEMU emulator:
+#
+MACHINE = "generic-armv8-xt"
```

Comment:
I'm not sure that `generic-armv8-xt` machine is able to work with renesas peripherals.

Reply:
well, in order to see proper machine names here we have to pull in meta-rcar-gen3+
at the same time, the root filesystem is really generic as it doesn't contain anything R-Car specific, but only kernel is